src/styles/product-card-styles/card.scss

```diff
@@ -0,0 +1,105 @@
+@import "variables.scss";
+
+.card {
+  margin: 0;
+  padding: 32px 16px 0;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  transition: all 300ms;
+
+  &__name {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+
+    .card__name {
+      color: #34568b;
+    }
+  }
+
+  &__photo {
+    margin: 0 3px 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $second-text-color;
+  }
+
+  &__code--position {
+    margin-top: 4px;
+  }
+
+  &__rate {
+    @include displayFlex;
+  }
+
+  &__rate--position {
+    margin-top: 16px;
+  }
+
+  &__review {
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    @include displayFlex;
+
+    &-description {
+      color: $second-text-color;
+      font-size: 12px;
+      line-height: 18px;
+    }
+
+    &-sum {
+      font-size: 16px;
+      line-height: 18px;
+      font-weight: 700;
+    }
+  }
+
+  &__price--position {
+    margin-top: 24px;
+  }
+
+  &__buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+
+    color: #fff;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-decoration: none;
+    text-transform: uppercase;
+
+    background-color: $active-color;
+    border: none;
+    border-radius: 5px;
+  }
+
+  &__buy:hover {
+    color: $active-color;
+    background-color: #fff;
+    border: 1px solid $active-color;
```

Comment:
You can add the transparent border for the default state and just change its color on hover